src/Mondu/Mondu/Controllers/OrdersController.php

```diff
@@ -49,7 +49,8 @@ public function create(WP_REST_Request $request) {
       } else {
         $this->validate_checkout();
         if (wc_notice_count('error') === 0) {
-          $order = $this->mondu_request_wrapper->create_order();
+          $lang = $data['lang'] ?? null;
```

Comment:
**question:** Why did you decide to receive it as a data parameter and not getting it directly from our backend (also applying the filters and so on)? I curious to hear your opinion about it.

Reply:
Because during the post to our order_controller I couldn't find any information regarding the localization.
I think because of that the wpml_current_language also didn't work correctly. 
That's why I decided to get it when the checkout page loads.